zds/forum/models.py

```diff
@@ -232,6 +237,11 @@ def meta_description(self):
             return first_post.text
         return Topic.__remove_greetings(first_post)[: settings.ZDS_APP["forum"]["description_size"]]
 
+    @property
+    def last_update(self):
+        last_visible_post = self.get_last_visible_post()
+        return last_visible_post.update or last_visible_post.pubdate
```

Comment:
Prenons le cas de deux sujets : 

- Sujet A dont le dernier message est il y a 10 heures ;
- Sujet B dont le dernier message est il y a 11 heures mais modifié il y a 9 heures.

Actuellement en production on affiche Sujet A puis Sujet B, mais sur ta PR on affiche Sujet B puis Sujet A. Pour afficher Sujet A puis Sujet B comme en production, il faut faire ce petit changement : 

```suggestion
        return self.get_last_visible_post().pubdate
```

Reply:
Ok, j'avoue ne pas avoir vérifier en production, le comportement me paraissait logique que la modification fasse remonter le topic (et d'ailleurs, tu te contredis ^^ : https://github.com/zestedesavoir/zds-site/pull/6056#discussion_r614725723).
Je prends note et je corrige !

Reply:
TODO : Rajouter un test pour gérer ce cas spécifiquement. Mon commit n'aurait pas dû casser un comportement existant. Je peux le rajouter dans cette PR, même si ça déborde un peu ?